### PR TITLE
Add recipe for pdpbox

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pdpbox/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (xhochy) [inline recipes/pdpbox/LICENSE.txt]: Made an upstream PR about this: https://github.com/SauceCat/PDPbox/pull/54

**Comment by xhochy** [comment]
@conda-forge/help-python Any idea why this simple recipe fails on OSX and Linux? This is working locally for me on OSX.

**Comment by scopatz** [comment]
Seems like it should work, restarting

**Comment by xhochy** [comment]
@isuruf 🤦‍♂ Thanks!

**Pull request merged**